--- origsrc/qtmultimedia-opensource-src-5.1.0/src/gsttools/gsttools.pro	2013-07-02 02:10:25.000000000 -0500
+++ src/qtmultimedia-opensource-src-5.1.0/src/gsttools/gsttools.pro	2013-08-16 18:29:16.848668600 -0500
@@ -44,7 +44,6 @@ PRIVATE_HEADERS += \
     qgstreamervideorendererinterface_p.h \
     qgstreameraudioinputselector_p.h \
     qgstreamervideorenderer_p.h \
-    qgstreamervideoinputdevicecontrol_p.h \
     gstvideoconnector_p.h \
     qgstcodecsinfo_p.h \
     qgstreamervideoprobecontrol_p.h \
@@ -60,13 +59,19 @@ SOURCES += \
     qgstreamervideorendererinterface.cpp \
     qgstreameraudioinputselector.cpp \
     qgstreamervideorenderer.cpp \
-    qgstreamervideoinputdevicecontrol.cpp \
     qgstcodecsinfo.cpp \
     gstvideoconnector.c \
     qgstreamervideoprobecontrol.cpp \
     qgstreameraudioprobecontrol.cpp \
     qgstreamervideowindow.cpp
 
+linux-* {
+    PRIVATE_HEADERS += \
+        qgstreamervideoinputdevicecontrol_p.h
+    SOURCES += \
+        qgstreamervideoinputdevicecontrol.cpp
+}
+
 qtHaveModule(widgets) {
     QT += multimediawidgets
 
--- origsrc/qtmultimedia-opensource-src-5.1.0/src/plugins/gstreamer/gstreamer.pro	2013-07-02 02:10:25.000000000 -0500
+++ src/qtmultimedia-opensource-src-5.1.0/src/plugins/gstreamer/gstreamer.pro	2013-08-16 18:37:00.441184600 -0500
@@ -2,11 +2,14 @@
 
 SUBDIRS += \
     audiodecoder \
-    mediacapture \
     mediaplayer
 
-config_gstreamer_encodingprofiles {
-    SUBDIRS += camerabin
+linux-* {
+    SUBDIRS += mediacapture
+
+    config_gstreamer_encodingprofiles {
+        SUBDIRS += camerabin
+    }
 }
 
 OTHER_FILES += \
