--- origsrc/qtmultimedia-opensource-src-5.1.0/src/gsttools/gsttools.pro	2013-07-02 02:10:25.000000000 -0500
+++ src/qtmultimedia-opensource-src-5.1.0/src/gsttools/gsttools.pro	2013-08-16 18:29:16.848668600 -0500
@@ -44,7 +44,6 @@ PRIVATE_HEADERS += \
     qgstreamervideorendererinterface_p.h \
     qgstreameraudioinputselector_p.h \
     qgstreamervideorenderer_p.h \
-    qgstreamervideoinputdevicecontrol_p.h \
     gstvideoconnector_p.h \
     qgstcodecsinfo_p.h \
     qgstreamervideoprobecontrol_p.h \
@@ -60,12 +59,18 @@ SOURCES += \
     qgstreamervideorendererinterface.cpp \
     qgstreameraudioinputselector.cpp \
     qgstreamervideorenderer.cpp \
-    qgstreamervideoinputdevicecontrol.cpp \
     qgstcodecsinfo.cpp \
     gstvideoconnector.c \
     qgstreamervideoprobecontrol.cpp \
     qgstreameraudioprobecontrol.cpp \
 
+linux-* {
+    PRIVATE_HEADERS += \
+        qgstreamervideoinputdevicecontrol_p.h
+    SOURCES += \
+        qgstreamervideoinputdevicecontrol.cpp
+}
+
 config_xvideo {
     DEFINES += HAVE_XVIDEO
 
--- origsrc/qtmultimedia-opensource-src-5.1.0/src/plugins/gstreamer/audiodecoder/qgstreameraudiodecoderserviceplugin.cpp	2013-07-02 02:10:25.000000000 -0500
+++ src/qtmultimedia-opensource-src-5.1.0/src/plugins/gstreamer/audiodecoder/qgstreameraudiodecoderserviceplugin.cpp	2013-08-16 18:36:21.470955600 -0500
@@ -49,7 +49,6 @@
 #include <QtCore/QDir>
 #include <QtCore/QDebug>
 
-#include <linux/types.h>
 #include <sys/time.h>
 #include <sys/ioctl.h>
 #include <sys/poll.h>
@@ -59,7 +59,6 @@
 #include <string.h>
 #include <stdlib.h>
 #include <sys/mman.h>
-#include <linux/videodev2.h>
 #include <gst/gst.h>
 
 // #define QT_SUPPORTEDMIMETYPES_DEBUG
--- origsrc/qtmultimedia-opensource-src-5.1.0/src/plugins/gstreamer/gstreamer.pro	2013-07-02 02:10:25.000000000 -0500
+++ src/qtmultimedia-opensource-src-5.1.0/src/plugins/gstreamer/gstreamer.pro	2013-08-16 18:37:00.441184600 -0500
@@ -2,11 +2,14 @@
 
 SUBDIRS += \
     audiodecoder \
-    mediacapture \
     mediaplayer
 
-config_gstreamer_encodingprofiles {
-    SUBDIRS += camerabin
+linux-* {
+    SUBDIRS += mediacapture
+
+    config_gstreamer_encodingprofiles {
+        SUBDIRS += camerabin
+    }
 }
 
 OTHER_FILES += \
--- origsrc/qtmultimedia-opensource-src-5.1.0/src/plugins/gstreamer/mediaplayer/qgstreamerplayerserviceplugin.cpp	2013-07-02 02:10:25.000000000 -0500
+++ src/qtmultimedia-opensource-src-5.1.0/src/plugins/gstreamer/mediaplayer/qgstreamerplayerserviceplugin.cpp	2013-08-16 18:35:46.389949100 -0500
@@ -51,7 +51,6 @@
 #include "qgstreamerplayerservice.h"
 #include <private/qgstutils_p.h>
 
-#include <linux/types.h>
 #include <sys/time.h>
 #include <sys/ioctl.h>
 #include <sys/poll.h>
@@ -61,7 +60,6 @@
 #include <string.h>
 #include <stdlib.h>
 #include <sys/mman.h>
-#include <linux/videodev2.h>
 #include <gst/gst.h>
 
 
