--- origsrc/qtmultimedia-opensource-src-5.1.0/tests/auto/integration/multimedia.pro	2013-07-02 02:10:25.000000000 -0500
+++ src/qtmultimedia-opensource-src-5.1.0/tests/auto/integration/multimedia.pro	2013-08-16 00:08:39.798565400 -0500
@@ -5,11 +5,12 @@ SUBDIRS += \
     qaudiodeviceinfo \
     qaudioinput \
     qaudiooutput \
-    qdeclarativevideooutput \
-    qdeclarativevideooutput_window \
     qmediaplayerbackend \
     qcamerabackend \
     qsoundeffect \
     qsound
 
 !qtHaveModule(widgets): SUBDIRS -= qcamerabackend
+qtHaveModule(qml): SUBDIRS += \
+    qdeclarativevideooutput \
+    qdeclarativevideooutput_window
--- origsrc/qtmultimedia-opensource-src-5.1.0/tests/auto/unit/multimedia.pro	2013-07-02 02:10:24.000000000 -0500
+++ src/qtmultimedia-opensource-src-5.1.0/tests/auto/unit/multimedia.pro	2013-08-16 00:04:13.523335300 -0500
@@ -30,7 +30,6 @@ SUBDIRS += \
     qvideosurfaceformat \
     qwavedecoder \
     qaudiobuffer \
-    qdeclarativeaudio \
     qaudiodecoder \
     qaudioprobe \
     qvideoprobe \
@@ -40,3 +39,6 @@ disabled {
     SUBDIRS += \
         qdeclarativevideo
 }
+
+qtHaveModule(qml):SUBDIRS += qdeclarativeaudio
+
